src/style.css

```diff
@@ -0,0 +1,127 @@
+:root {
+  --main_text: #060b35;
+  --secondary_text: #616070;
+  --design_color: #00acdc;
+  --background_color_buy: #fff;
+  --border_color: 1px solid var(--design_color);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: var(--background_color_buy);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__img-imac {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 0 auto 40px;
+}
+
+.card__about {
+  font-weight: 500;
+  line-height: 18px;
+  font-size: 12px;
+  color: var(--main_text);
+  margin: 0 0 4px;
+}
+
+.card__id {
+  margin: 0 0 17px;
```

Comment:
To reset default values you can use tag selectors `p, h1, h2, h3... { margin: 0 }` Please fix for all cases where you reset default margin of `p` tag

Reply:
https://github.com/mate-academy/layout_product-cards/pull/2890/files#r1170523329
fix this comment